patches/net/minecraft/world/item/ItemStack.java.patch

```diff
@@ -9,17 +9,23 @@
     public static final Codec<ItemStack> CODEC = RecordCodecBuilder.create(
        p_258963_ -> p_258963_.group(
                 BuiltInRegistries.ITEM.byNameCodec().fieldOf("id").forGetter(ItemStack::getItem),
-@@ -86,6 +_,9 @@
+@@ -86,6 +_,7 @@
              )
              .apply(p_258963_, ItemStack::new)
     );
-+   @org.jetbrains.annotations.Nullable
-+   private final net.minecraft.core.Holder.Reference<Item> delegate;
 +   private CompoundTag capNBT;
     private static final Logger LOGGER = LogUtils.getLogger();
     public static final ItemStack EMPTY = new ItemStack((Void)null);
     public static final DecimalFormat ATTRIBUTE_MODIFIER_FORMAT = Util.make(
-@@ -140,27 +_,39 @@
+@@ -107,7 +_,6 @@
+    private static final Style LORE_STYLE = Style.EMPTY.withColor(ChatFormatting.DARK_PURPLE).withItalic(true);
+    private int count;
+    private int popTime;
+-   @Deprecated
```

Comment:
This should be added back. :P

Reply:
Oh I thought that was added by Forge - something something reg replacement.